jsk_kinova_robot/jsk_kinova_description/robots/gen3_robotiq_2f_140.xacro

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```

Comment:
is this copy of  https://github.com/Kinovarobotics/ros_kortex/blob/melodic-devel/kortex_description/robots/gen3_robotiq_2f_140.xacro ??

Reply:
Yes. this is copied. may be orignal model should not be directly written, but loaded.

Reply:
License Descriptions are added